src/sonic-yang-models/yang-models/sonic-bgp-global.yang

```diff
@@ -445,6 +445,11 @@ module sonic-bgp-global {
                     }
                     description "Maximum duration to suppress a stable route";
                 }
+
+                leaf advertise-all-vni {
```

Comment:
Thanks for the bugfix! Could you also add a unit test, in order to prevent future regression?

Reply:
I guess, but there are currently ZERO tests for BGP_DEVICE_GLOBAL_AF.  Seems odd for this to be the only test.